src/cpp/shared/combat/animation.cpp

```diff
@@ -346,24 +346,24 @@ SCmbtHero sCmbtHero[13] = {
 
   // Cyborg
   {
```

Comment:
You overwrote the actual Cyborg hero graphics. The captain should be at index 24, right?

Reply:
Why 24? I put it here because sCmbtHere had originally size of 6 (not enough space for captains obviously)

Reply:
Captains are at factionId + 6, right? Isn't FACTION_CYBORG equal to 18?

Oh wait, it's 12. Then you should move the values of the Cyborg SCmbtHero to the right index, and test that cyborg heroes still work in battle.

Reply:
What's the right index? sCmbtHero has a size of MAX_FACTIONS and captains can't fit there unless I heavily misunderstand something

Reply:
The right index would be index 12, right?

Reply:
Yes. It is on the index 12 now

Reply:
Alright HEROES3W.c confused me big time. It's defined there as sCmbtHero[6] NOT as sCmbtHero[12] which is the correct size.